src/app/screens/BoardsList/Components/AddBoardModal/layout.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Modal from 'react-awesome-modal';
+import { Field, reduxForm } from 'redux-form';
+
+import './styles.css';
+
+const AddBoardModal = ({ handleSubmit, visible, closeModal, openModal }) => (
```

Comment:
Mmm this is name is a bit confuse. It isn't a Modal at all, it is a button that spawns a modal.
What about renaming it to `AddBoardForm` ?

Reply:
It's actually the modal